pages/measurement-sets/[id].js

```diff
@@ -169,6 +153,29 @@ export default function MeasurementSet({
       : [];
   const uniqueSampleSummaries = [...new Set(sampleSummaries)];
 
+  // Collect all measurement set protocols
+  let measurementSetProtocols = [];
+  if (measurementSet.protocols && measurementSet.protocols.length > 0) {
+    measurementSetProtocols = measurementSet.protocols;
+  }
```

Comment:
I think this is easier to read as:
```
const measurementSetProtocols = measurementSet.protocols || [];
```
Or not even have this at all if you use the suggestion below.

Reply:
i removed this